### PR TITLE
graph_msgs: 0.1.0-2 in 'noetic/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by JafarAbdi**
Increasing version of package(s) in repository `graph_msgs` to `0.1.0-2`:

- upstream repository: https://github.com/PickNikRobotics/graph_msgs.git
- release repository: https://github.com/PickNikRobotics/graph_msgs-release.git
- distro file: `noetic/distribution.yaml`
- bloom version: `0.9.7`
- previous version for package: `null`

## graph_msgs

```
* Added header / timestamp
* Contributors: Dave Coleman
```

### CODE REVIEW

**Comment by JafarAbdi** [comment]
@sloretz Can I now release a package that depends on this one .? or should I wait until the sync is out and release it with the next sync .?

I'm getting this

```
Could not resolve rosdep key 'graph_msgs'
Failed to resolve graph_msgs on ubuntu:focal with: Error running generator: Failed to resolve rosdep key 'graph_msgs', aborting.
graph_msgs is depended on by these packages: ['rviz_visual_tools']
<== Failed
Some of the dependencies for packages in this repository could not be resolved by rosdep.
```

**Comment by sloretz** [comment]
holding for the [upcoming Noetic sync](https://discourse.ros.org/t/preparing-for-noetic-sync-2020-06-05/14451)

> Can I now release a package that depends on this one .? or should I wait until the sync is out and release it with the next sync .?

@JafarAbdi I'm not sure. It might need to wait until this PR is merged; which means after the sync Tomorrow.

File: noetic/distribution.yaml
Comment (clalancette) [inline noetic/distribution.yaml:890]: The previous versions of this package used the URL as https://github.com/davetcoleman/graph_msgs.git .  It's OK to move them, but I would like to see:

a)  That @davetcoleman approves of this, and
b)  Updating Melodic and Kinetic to point to this repository

**Review by clalancette — DISMISSED** [review]

File: noetic/distribution.yaml
Comment (davetcoleman) [reply noetic/distribution.yaml:890]: I approve. PickNik is the company we founded :-)

**Review by davetcoleman — APPROVED** [review]

**Review by sloretz — APPROVED** [review]

File: noetic/distribution.yaml
Comment (clalancette) [reply noetic/distribution.yaml:949]: Thanks, appreciated.

**Pull request merged**
